api/app/settings/logger.py

```diff
@@ -0,0 +1,22 @@
+import json
+import logging
+
+
+class JsonFormatter(logging.Formatter):
+    """Custom formatter for json logs."""
+
+    def format(self, record):
+        try:
+            log_message = record.msg.replace('"', "'").replace("%s", "{}")
+            formatted_message = log_message.format(*record.args)
+            log_record = {
+                "levelname": record.levelname,
+                "message": formatted_message,
+                "timestamp": self.formatTime(record, self.datefmt),
+                "logger_name": record.name,
+                "process_id": record.process,
+                "thread_name": record.threadName,
+            }
+            return json.dumps(log_record)
+        except Exception as e:
```

Comment:
Is it normal in a logging formatter to just catch all exceptions or can we be more explicit here?

Reply:
Good point, was just trying to avoid an unhandled exception/app crash due to logging formatting -- catching the json encoding error is likely good enough.  I'll take a look at the possible exceptions cases and narrow it down.